### ISSUE TITLE
S3220 (InvocationResolvesToOverrideWithParams) has FP on Console.WriteLine()

### ISSUE THREAD

**Issue by dbolkensteyn** [SonarSource-VisualStudio/sonarlint-roslyn-analyzer#750]
The following code triggers an issue of S3220, which is a false positive:
```
Console.WriteLine("format", 0, 1, "", "")
```
